modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
+
+Autoscaling is dependent on the resource quotas that are set. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
+
+[NOTE]
+====
+Autoscaling can be disabled on a machine pool, but it cannot be disabled on a cluster. However, if no machine pool autoscaling is enabled, then autoscaling will not occur on that cluster even if autoscaling is enabled on the cluster.
```

Comment:
I am curious why we have to say this at all.  Seems like it will invite questions for no reason.

Reply:
It was added based on comments in Oren's Google Doc. I'm removing it based on your feedback.